web/src/app/modules/shared/shared.module.ts

```diff
@@ -92,8 +92,10 @@ import { IconComponent } from './components/presentation/icon/icon.component';
 import { FormViewContainerComponent } from './components/form-view-container/form-view-container.component';
 import { SignpostComponent } from './components/presentation/signpost/signpost.component';
 import { JoinPipe } from './pipes/join/join.pipe';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
```

Comment:
This is not needed

Reply:
Unfortunately it's needed temporarily as a workaround for [see Clarity issue](https://github.com/vmware/clarity/issues/6080#issuecomment-862742168)